### ISSUE TITLE
tensorflow2.0 FusedBatchNormV3 转换问题 

### ISSUE THREAD

**Issue by xinyidaren** [kendryte/nncase#106]
平台：tensorflow2.0
出现问题函数：tf.keras.layers.BatchNormalization()
问题描述：训练模型中出现FusedBatchNormV3时，ncc转换时并没有报错，但在使用中出现g_ai_done_flag不等于1的问题

以下时代码截取:

模型生成代码:
import tensorflow as tf
from tensorflow.keras import layers
from tensorflow.keras.models import Model
import os

input_image = tf.keras.layers.Input(shape=[128, 128, 3], name="input_image")
center = layers.Conv2D(8,(3,3),padding='same',activation=tf.nn.relu6)(input_image)
center = layers.BatchNormalization()(center) #参数training取true和false都不好使
center = layers.MaxPooling2D(pool_size=[2,2],strides=[2,2],padding='SAME')(center)
center = layers.Conv2D(8,(3,3),padding='same',activation=tf.nn.relu6)(center)
center = layers.MaxPooling2D(pool_size=[2,2],strides=[2,2],padding='SAME')(center)
center = layers.Conv2D(8,(3,3),padding='same',activation=tf.nn.relu6)(center)
center = layers.MaxPooling2D(pool_size=[2,2],strides=[2,2],padding='SAME')(center)
center = layers.Conv2D(8,(3,3),padding='same',activation=tf.nn.relu6)(center)
center = layers.MaxPooling2D(pool_size=[2,2],strides=[2,2],padding='SAME')(center)
center = layers.Flatten()(center)
center = layers.Dense(16, activation='relu')(center)
center = layers.Dense(10, activation='softmax')(center)
model = Model(inputs=[input_image], outputs=[center])

converter = tf.lite.TFLiteConverter.from_keras_model(model)
tflite_model = converter.convert()
if not os.path.exists('./tflite_models'):
    os.mkdir('./tflite_models')
with open('./tflite_models/keras.tflite', 'wb') as f:
    f.write(tflite_model)
# 已做测试，与模型输入大小无关，在模型不包含ayers.BatchNormalization()时，其他算子构成的模型无问题。

模型转换代码（ncc版本为NNCase v0.2.0 Beta2）：
./ncc compile keras.tflite keras.kmodel -i tflite -o kmodel -t k210 --dataset ./images

模型调用部分代码：

    g_ai_done_flag = 0;
    kpu_run_kmodel(&task, kpu_image.addr, DMAC_CHANNEL5, ai_done,
                   NULL);

    while (!g_ai_done_flag)

调用发现g_ai_done_flag恒为0

p.s.:当转换时代码为./ncc compile keras.tflite keras.kmodel -i tflite -o kmodel -t k210 --inference-type float./ncc compile keras.tflite keras.kmodel -i tflite -o kmodel -t k210 --inference-type float时
居然可以用，就是慢

**Comment by sunnycase** [comment]
你能附上出问题的 tflite 吗？

**Comment by sunnycase** [comment]
Close as inacive.
